External/Plugins/HaXeContext/Context.cs

```diff
@@ -388,6 +388,8 @@ public override void BuildClassPath()
             // add haxe libraries
             if (proj != null)
             {
+                proj.UpdateVars(false);
```

Comment:
This can have serious consequences, especially with OpenFL-like projects types.

We should just make sure that pressing the 'Refresh' button in the project panel does resolve the haxelibs properly - it normally should be the case.